docs/guides/update-guide/elasticsearch/7-to-8.md

```diff
@@ -0,0 +1,19 @@
+---
+id: 7-to-8
+title: Update Elasticsearch 7 to 8
```

Comment:
🔧 To align with the other Update Guides, lets change this title to only the version numbers
```suggestion
title: Update 7 to 8
```

Reply:
I prefer keeping it like this so people don't confuse this page with C7 and C8 😅

Reply:
I don't agree. The Connectors also have their own version numbering.

Reply:
<img width="320" alt="Screenshot 2023-04-06 at 17 07 44" src="https://user-images.githubusercontent.com/3511026/230421108-23e75bb1-afdc-485a-9a95-9f986cf02cd5.png">